packages/server/src/plugin/subscriptionCallback/index.ts

```diff
@@ -213,12 +220,11 @@ class SubscriptionManager {
         interval: NodeJS.Timeout;
         queue: Promise<void>[];
       };
-      subscriptionsById: Map<string, SubscriptionObject>;
+      subscription?: SubscriptionObject;
     }
   > = new Map();
```

Comment:
I don't understand what you would like to have here. We still have to keep that map because we have to track every subscriptions so... I think I'm confused. I'm updating the comment though

Reply:
I think I may have misunderstood something. In the doc I saw patterns like this:
`http://localhost:4000/callback/c4a9d1b8-dc57-44ab-9e5a-6e6189b2b945`

I figured the `/callback` path was standard but it seems it's configurable, which means tracking the full URL is probably simplest as you've noted elsewhere in a comment below.